src/Components/Search.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Input } from 'antd';
+
+const Search = (props) => {
+
+
+    return(
+        <div className="search-bar">
+            <Input.Search
+                placeholder="input search text"
+                value={props.input}
+                onChange={props.updateInput}
```

Comment:
A nice enhancement would be to return to full, unfiltered results when the search term is deleted, aka input is undefined.